Xamarin.Forms.Platform.Android/AndroidTicker.cs

```diff
@@ -40,12 +42,19 @@ internal void CheckPowerSaveModeStatus()
 			var powerSaveOn = powerManager.IsPowerSaveMode;
 
 			// If power saver is active, then animations will not run
-			_systemEnabled = !powerSaveOn;
+			_energySaveModeDisabled = !powerSaveOn;
 			
 			// Notify the ticker that this value has changed, so it can manage animations in progress
 			OnSystemEnabledChanged();
 		}
 
+		static bool IsAnimatorEnabled()
+		{
+			var resolver = global::Android.App.Application.Context.ContentResolver;
+			var scale = global::Android.Provider.Settings.Global.GetFloat(resolver, global::Android.Provider.Settings.Global.AnimatorDurationScale, 0);
```

Comment:
Could any of this be null? Do we need some error handling here?

Reply:
@jfversluis Should we consider to turn off animations if `Context` or `ContextResolver` is null?

By using `false` as default, we would make sure that the app stays usable.